### PR TITLE
Added steps to run oracle database 23c free on Mac M1 chip in FAQ.md

### PR DESCRIPTION
**Pull request by aditya059**
Added steps to run oracle database 23c free on Mac M1 chip in FAQ.md

### CODE REVIEW

File: OracleDatabase/SingleInstance/FAQ.md [hunk 1]
```diff
@@ -110,3 +114,50 @@ Refer to the [module documentation](https://python-oracledb.readthedocs.io/en/la
...
+6. Check whether the database came up healthy.
+
+    ```sh
+    $ podman ps -a
+        CONTAINER ID  IMAGE                                               COMMAND               CREATED        STATUS                  PORTS                    NAMES 
+        825dcbd3e822  container-registry.oracle.com/database/free:latest  /bin/sh -c exec $...  4 minutes ago  Up 4 minutes (healthy)  0.0.0.0:42439->1521/tcp  hopeful_yalow
+    ```
+
+### Connect
+We need to get a Java based SQL client on Apple M1 due to unavailability of native SQL client.
```
Comment (cjbj) [inline OracleDatabase/SingleInstance/FAQ.md:154]: It's not true that you need to get a Java based SQL client.  This statement needs to be fixed before merging.

You can simply install the Intel Instant Client and Rosetta does the rest automatically when you invoke SQL*Plus.  The install instructions are the same on ARM as for Intel. You just need to install/authorize Rosetta.

Comment (cjbj) [reply OracleDatabase/SingleInstance/FAQ.md:154]: The quickest way to install SQL*Plus is via the command line:
```
cd $HOME/Downloads
curl -O https://download.oracle.com/otn_software/mac/instantclient/198000/instantclient-basic-macos.x64-19.8.0.0.0dbru.dmg
curl -O https://download.oracle.com/otn_software/mac/instantclient/198000/instantclient-sqlplus-macos.x64-19.8.0.0.0dbru.dmg
hdiutil mount instantclient-basic-macos.x64-19.8.0.0.0dbru.dmg
hdiutil mount instantclient-sqlplus-macos.x64-19.8.0.0.0dbru.dmg
/Volumes/instantclient-basic-macos.x64-19.8.0.0.0dbru/install_ic.sh
hdiutil unmount /Volumes/instantclient-basic-macos.x64-19.8.0.0.0dbru
hdiutil unmount /Volumes/instantclient-sqlplus-macos.x64-19.8.0.0.0dbru
```

**Review by cjbj — CHANGES_REQUESTED** [review]

File: OracleDatabase/SingleInstance/FAQ.md (hunk 1 above)
Comment (aditya059) [reply OracleDatabase/SingleInstance/FAQ.md]: Hi @cjbj 

Updated.

Thanks

File: OracleDatabase/SingleInstance/FAQ.md [hunk 2]
```diff
@@ -110,3 +114,65 @@ Refer to the [module documentation](https://python-oracledb.readthedocs.io/en/la
 ## ORA-01157: cannot identify/lock data file
 
 This error occurs when the database cannot find a data file (used for tablespaces) that was previously present. This is most likely because the data file has been located outside the volume in a previous container and was hence not persisted. Ensure that when you add tablespaces and/or data files that they are located within the volume location, i.e. $ORACLE_BASE/oradata/$ORACLE_SID, (e.g. `/opt/oracle/oradata/XE`).
+
+## Running Oracle Database 23c Free on Apple based ARM M1 chip
```
Comment (cjbj) [inline OracleDatabase/SingleInstance/FAQ.md:118]: This probably applies to M2 as well. Perhaps use a less version-specific title?  Apple Silicon?  Apple ARM?  IDK.

File: OracleDatabase/SingleInstance/FAQ.md [hunk 3]
```diff
@@ -110,3 +114,65 @@ Refer to the [module documentation](https://python-oracledb.readthedocs.io/en/la
...
+
+   ```/Volumes/instantclient-basic-macos.x64-19.8.0.0.0dbru/install_ic.sh```
+
+   ```hdiutil unmount /Volumes/instantclient-basic-macos.x64-19.8.0.0.0dbru```
+
+   ```hdiutil unmount /Volumes/instantclient-sqlplus-macos.x64-19.8.0.0.0dbru```
+
+2. Test the connection:
+
+    ```$HOME/Downloads/instantclient_19_8/sqlplus sys/<ORACLE_PWD>@localhost:<PORT>/FREE as sysdba```
```
Comment (cjbj) [inline OracleDatabase/SingleInstance/FAQ.md:174]: - For security reasons, don't include the password in the command line.  SQL*Plus will prompt for it.

- I don't know the Oracle DB Dockerfile standard, but for everyday use it is more useful to show connecting to the PDB than to the CDB.

- Some users may prefer sqlcl so you could offer those steps as an alternative, depending how much complexity you want to add in this FAQ.

**Review by cjbj — CHANGES_REQUESTED** [review]

File: OracleDatabase/SingleInstance/FAQ.md (hunk 3 above)
Comment (aditya059) [reply OracleDatabase/SingleInstance/FAQ.md]: Fixed. Thanks

**Review by yunus-qureshi — APPROVED** [review]

**Review by gvenzl — APPROVED** [review]

**Comment by aditya059** [comment]
Hi @gvenzl 
Please merge the changes. Thanks

**Review by cjbj — APPROVED** [review]
I have no further comments on the bits that concerned me.

**Pull request merged**